lib/language_pack/ruby.rb

```diff
@@ -924,13 +776,12 @@ def attribute(name, value, force_string = false)
   <%= key %>: <%= value.first %>
 <% end %>
         DATABASE_YML
-      end
     end
   end
 
   def rake
     @rake ||= begin
-      rake_gem_available = bundler.has_gem?("rake") || ruby_version.rake_is_vendored?
+      rake_gem_available = true
```

Comment:
Since this is now always `true`, is this variable even necessary? Is there code not already accounted for this PR that depends on this boolean for conditional logic?